### PR TITLE
DN-5378 firefox extension

### PR DESCRIPTION
**Pull request by mjbradford89**
The add on has not yet been approved for firefox, but there are other improvements on this branch as well that I would like to get into master.

### CODE REVIEW

**Review by alanjds — APPROVED** [review]

File: firefox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Earn NCT for contributing cyber security data",
+  "version": "1.0.8",
+  "manifest_version": 2,
+  "name": "PolySwarm in Browser",
```
Comment (mrtizmoatwork) [inline firefox/manifest.json:5]: let's go with our new name and descriptions

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ so for now, users have to work with the .tar.gz files we release here on GitHub.
 
 ## Installation
 
-1. Download the [NectarNet Beta Browser Extension](https://github.com/polyswarm/ppdns-chrome-extension/releases/download/v1.0.8/ppdns-chrome-extension-v1.0.8.zip) .zip file.
-2. Double-click the `ppdns-chrome-extension-v1.0.8-zip` file to unzip it.
+1. Download the [NectarNet Beta Browser Extension](https://github.com/polyswarm/ppdns-chrome-extension/releases/download/v1.0.9/ppdns-chrome-extension-v1.0.9.zip) .zip file.
```
Comment (mrtizmoatwork) [inline README.md:18]: let's add 2 sections of Installatiion

1. Manual Installation (<- this is using the .zip)
2. Installation from Extension Store (<- this is from the browser store)

Comment (mjbradford89) [reply README.md:23]: updated in [44a153b](https://github.com/polyswarm/ppdns-chrome-extension/pull/23/commits/44a153b7599582851094bb1edf3dad1e86a25394)

File: firefox/manifest.json (hunk 1 above)
Comment (mjbradford89) [reply firefox/manifest.json]: updated in [82e6cea](https://github.com/polyswarm/ppdns-chrome-extension/pull/23/commits/82e6cea5b108ffaf7d207006e0e6c9603622b6ff)

**Pull request merged**